tests/python/pants_test/tasks/test_junit_tests_integration.py

```diff
@@ -170,7 +170,8 @@ def test_junit_test_requiring_cwd_passes_with_option_with_no_value_specified(sel
         '--interpreter=CPython>=2.6,<3',
         '--interpreter=CPython>=3.3',
         '--jvm-test-junit-options=-Dcwd.test.enabled=true',
-        '--test-junit-cwd',])
+        '--test-junit-cwd',
```

Comment:
You mean if `--test-junit-cwd` were set to something other than None/emptystring? The `--test-junit-default-to-first-target` flag would have no effect, because the "default" value doesn't matter if it's explicitly set.